extensions/functions_logarithmic.yaml

```diff
@@ -102,5 +102,29 @@ scalar_functions:
             name: "base"
             description: "The logarithm base `b` to use"
         return: fp64
+  -
+    name: "log1p"
+    description: >
+      Logarithm of 1 + the value
 
+      log1p(x) => log(1+x)
```

Comment:
What is the base of the `log` here?

Reply:
Have updated the description with more detail.